common/lib/xmodule/xmodule/js/src/video/06_video_progress_slider.js

```diff
@@ -147,25 +149,6 @@ function () {
         // with actual starting and ending point of the video.
 
         rangeParams = getRangeParams(start, end, duration);
-
-        if (!this.videoProgressSlider.sliderRange) {
```

Comment:
The slider range was the original way start/stop times were shown - the entire video's time would be displayed, and the sliderRange had two stops connected by a red bar that indicated the start / end times. With hard start/stops, we no longer want to show the two stops and red bar.